layers/meta-tegrademo/recipes-bsp/u-boot/u-boot-tegra/0015-t210-mender-FIT-image-bootcmd-support.patch

```diff
@@ -0,0 +1,107 @@
+From d1428fef6bd84bc1c95351cb179178c60011869b Mon Sep 17 00:00:00 2001
+From: Ilies CHERGUI <ilies.chergui@gmail.com>
+Date: Sun, 17 Jan 2021 19:34:18 +0000
+Subject: [PATCH] tegra210: mender FIT image bootcmd support.
+
+Signed-off-by: Ilies CHERGUI <ilies.chergui@gmail.com>
+---
+ include/configs/tegra210-common.h |  7 +++++--
+ include/env_default.h             |  7 +++++--
+ include/env_mender.h              | 18 ++++++++++++++++++
+ scripts/config_whitelist.txt      |  1 +
+ 4 files changed, 29 insertions(+), 4 deletions(-)
+
+diff --git a/include/configs/tegra210-common.h b/include/configs/tegra210-common.h
+index 35cead04de..ab38a7e44f 100644
+--- a/include/configs/tegra210-common.h
++++ b/include/configs/tegra210-common.h
+@@ -41,6 +41,8 @@
+  *   for the FDT/DTB to be up to 1M, which is hopefully plenty.
+  */
+ #define CONFIG_LOADADDR 0x80080000
++/* FIT Image Load address */
++#define CONFIG_FITFDT_LOADADDR 0x90000000
+ #define MEM_LAYOUT_ENV_SETTINGS \
+ 	"scriptaddr=0x90000000\0" \
+ 	"pxefile_addr_r=0x90100000\0" \
+@@ -81,8 +83,9 @@
+ 		"/chosen/uuid:" \
+ 		"/chosen/linux,initrd-start:" \
+ 		"/chosen/linux,initrd-end:" \
+-		"/psci/nvidia,system-lp0-disable:" \
+-		"/serial-number\0"
++		"/serial-number\0" \
++	"fitfdt_loadaddr=" __stringify(CONFIG_FITFDT_LOADADDR) "\0"
```

Comment:
I don't understand why there's a separate setting for this - other platforms don't appear to need this addition.

If it's really needed, it looks like it would be better to cluster it with the other address-related settings, above the FDT fixup definitions.

Reply:
Right, I will try to update it